libraries/Microsoft.Bot.Builder.Ai.QnA/QnAMaker.cs

```diff
@@ -79,6 +79,16 @@ public QnAMaker(QnAMakerOptions options, HttpClient httpClient = null)
         /// <returns>A list of answers for the user query, sorted in decreasing order of ranking score.</returns>
         public async Task<QueryResult[]> GetAnswers(string question)
         {
+            if (_options.ScoreThreshold < 0 || _options.ScoreThreshold > 1)
```

Comment:
Should these be in the constructor, when the Options class is first passed in? 

It would be weird to get that exception here.